content/docs/atom-guide/look-dev/six-point-lighting-tutorial.md

```diff
@@ -0,0 +1,439 @@
+      surface.right = topLeftRightBottom.b;
+      surface.bottom = topLeftRightBottom.a;
+      surface.frontside = frontBack.r;
+      surface.backside = frontBack.g;
+   }
+   else
+   {
+      float4 rightLeftTop = MaterialSrg::m_rightLeftTopMap.Sample(MaterialSrg::m_sampler, sixPointUv);
+      float4 bottomBackFront = MaterialSrg::m_bottomBackFrontMap.Sample(MaterialSrg::m_sampler, sixPointUv);
+      surface.right = rightLeftTop.r;
+      surface.left = rightLeftTop.g;
+      surface.top = rightLeftTop.b;
+      surface.bottom = bottomBackFront.r;
+      surface.backside = bottomBackFront.g;
+      surface.frontside = bottomBackFront.b;
+   }
+   ```
+
+4. Initalize the `tangent` and `bitangent` surface properties right after:
+   ```glsl
+   surface.tangent = tangents[0];
+   surface.bitangent = bitangents[0];
+   ```
+
+## Add custom lighting
+Now that you have set up the six-point surface, you can use the new surface properties in custom lighting. You will make customize two types of lighting: directional lighting and image-based lighting (IBL). Directional lighting is lighting for if a light came from a direction, while IBL simulates omni-directional reflective, almost ambient, lighting from the environment around the entity.
+
+### Add custom directional lighting
+As discussed earlier, you will make a light map that uses the lighting direction to determine which combination of the six sides should be lit up. Then, you can take the intensity of the light and how much the texel should be lit up from lighting in that direction to compute the overall lighting on that particular texel.
+
+1. Open `SixPointLighting.azsli`.
+   
+   1. Notice the `#include <SixPointSurface.azsli>` line at the top. This is how you can reference the surface in the following functions.
+
+   2. Notice the functions `GetDiffuseLighting()` and `GetSpecularLighting()`. You will want to edit these two achieve the desired effects. 
+
+      The six-point lighting `ForwardPassPS_Common` shader uses the default `ApplyDirectLighting()` function, which will iterate over the lights that apply to this object and invoke these custom `GetDiffuseLighting()` and `GetSpecularLighting()` functions for each light.
+
+2. Start off editing the lighting simply with `GetSpecularLighting()`.
+   
+   Specular lighting simulates the bright spot on a shiny object that most brightly reflects light into the camera. In the six-point lighting case, you do not need any specular lighting because 1) specular lighting can't effectively be applied on the 2D textures and 2) the use cases do not require it, as you are mainly using this technique for clouds and non-shiny objects.
+
+   1. Edit the `GetSpecularLighting()` function to not return any specular lighting.
+   
+      ```glsl
+      float3 GetSpecularLighting(Surface surface, LightingData lightingData, const float3 lightIntensity, const float3 dirToLight)
+      {
+         return float3(0.0f, 0.0f, 0.0f);
+      }
+      ```
```

Comment:
Good points, I should add another sentence for this but this function is called by a lighting function in a different file so it doesn't appear anywhere else. The specular function is define in the templates file because that's the default implementation for StandardPBR; I'll add a sentence for this as well.

Reply:
Will this shader code still run if this function didn't exist? I'm wondering if specular lighting is used anywhere else in the six-point lighting material. If not, it might be best to remove it from the templates. Then you can remove it from this documentation as well.

Reply:
I see; the shader code unfortunately won't run if the function didn't exist. But if you think this part is unhelpful, I could just have the template file use the final specular lighting code that uses `return float3(0.0f, 0.0f, 0.0f);`? Then I could add a sentence about the specular lighting but otherwise get rid of this section of the tutorial.

Reply:
Yeah, if possible could you return (0,0,0) in the template instead? I don't think there's value in having it in the tutorial, since the material type the user's creating doesn't use specular lighting. If anything, it's informative to know that specular lighting doesn't apply to 2D textures well.

Reply:
Yes, will change the code for that & I placed the specular lighting explanation above when I said to look at the functions!